modules/vorbis/audio_stream_ogg_vorbis.h

```diff
@@ -133,6 +133,7 @@ class AudioStreamOggVorbis : public AudioStream {
 	double bpm = 0;
 	int beat_count = 0;
 	int bar_beats = 4;
+	Dictionary tags;
```

Comment:
@Calinou I think that we shouldn't use `Dictionary` internally. I'm I wrong?

Reply:
It's *fine* to use Dictionary internally, but `HashMap<String, Variant>` provides stronger typing guarantees.